src/__tests__/e2e/homepage_test.py

```diff
@@ -1,17 +1,21 @@
+"""Runs Selenium tests agains the homepage."""
```

Comment:
I think you accidentally wrote "agains" instead of "against" :)

Reply:
Oops! Yes!